web/src/pages/Settings/Settings.jsx

```diff
@@ -151,7 +151,7 @@ const Settings = ({ openPage }) => {
                                 <UpdateIcon fontSize="inherit" />
                             </Badge>
                         </div>
-                    Updates
+                    Admin Panel
```

Comment:
![Screenshot 2024-10-04 121440](https://github.com/user-attachments/assets/5c64c90a-c9b6-4750-8685-5dd1b85b7e16)
![Screenshot 2024-10-04 122056](https://github.com/user-attachments/assets/1c8f7a87-a7ca-4ed3-b3bd-f818a6f101cc)
Had to make a compromise to name this something accurate that also fits on the thinnest screens

Reply:
love this; it's official :sunglasses: